web/src/main/client/src/components/feed/Feed.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import ReactPaginate from 'react-paginate';
+
+import Article from '../../components/article/Article';
+import Tweet from '../../components/tweet/Tweet';
+import Quote from '../../components/quote/Quote';
+
+let PAGE_SIZE = 20;
+
+
+class Feed extends Component {
+
+  constructor(props) {
+    super(props);
+
+    //Initialize an empty state object
+    this.state = {};
+  }
+
+  componentDidMount() {
+    //Load first page when component mounts
+    this.loadItems(0);
+  }
+
+  //loadItems load
+  loadItems(pageNumber){
+    const url = `https://happynews-api.svendubbeld.nl/admin/posts?page=${pageNumber}&size=${PAGE_SIZE}`;
```

Comment:
It might be nice to put the URL in either a constant or a config file (preferably the latter), this makes it easier to configure and test locally.